src/features/machines/depositedBalanceMachine.ts

```diff
@@ -38,14 +42,14 @@ type ThisActor = ActorRef<Snapshot<unknown>, SharedEvents>
 
 export type Events =
   | { type: "LOGOUT" | "REQUEST_BALANCE_REFRESH" }
-  | { type: "LOGIN"; params: { accountId: string } }
+  | { type: "LOGIN"; params: { userAddress: string; userChainType: ChainType } }
 
 export const depositedBalanceMachine = setup({
   types: {
     context: {} as {
       parentRef: ParentActor
       defuseTokenIds: string[]
-      userAccountId: string | null
+      userAccountId: DefuseUserId | null
```

Comment:
Probably we should rename `userAccountId` to `useAddress`. The same as we do for just `accountId`.

Reply:
Yes, I deliberately skipped all the refactoring part, because it would be hard to review all changes.